.circleci/config.yml

```diff
@@ -537,14 +540,48 @@ workflows:
           name: build-metrics-server
           container-image: metrics-server
       - build-moco-mysql:
+          name: build-moco-mysql-8018
+          tag-directory: moco-mysql-8018
+          mysql-version: 8.0.18
+      - build-moco-mysql:
+          tag-directory: moco-mysql
           mysql-version: 8.0.20
```

Comment:
How about (re)naming this job "build-moco-mysql- **8020**"? (also following container image build job)